modular_zubbers/code/modules/customization/species/proteans/organs/protean_brain.dm

```diff
@@ -91,6 +91,8 @@
 	owner.invisibility = 101
 	new /obj/effect/temp_visual/protean_to_suit(owner.loc, owner.dir)
 	owner.Stun(INFINITY, TRUE)
+	owner.add_traits(list(TRAIT_RESISTLOWPRESSURE, TRAIT_RESISTHIGHPRESSURE, TRAIT_RESISTHEAT, TRAIT_RESISTCOLD), PROTEAN_TRAIT)
```

Comment:
mind using a list on the brain for the traits to add/remove?

Reply:
A define is probably a better idea since it's in only one location